pythia/utils/process_answers.py

```diff
@@ -5,244 +5,251 @@
+            "whyre": "why're",
+            "whys": "why's",
+            "wont": "won't",
+            "wouldve": "would've",
+            "wouldnt": "wouldn't",
+            "wouldnt've": "wouldn't've",
+            "wouldn'tve": "wouldn't've",
+            "yall": "y'all",
+            "yall'll": "y'all'll",
+            "y'allll": "y'all'll",
+            "yall'd've": "y'all'd've",
+            "y'alld've": "y'all'd've",
+            "y'all'dve": "y'all'd've",
+            "youd": "you'd",
+            "youd've": "you'd've",
+            "you'dve": "you'd've",
+            "youll": "you'll",
+            "youre": "you're",
+            "youve": "you've",
+        }
 
+        self.manual_map = {
+            "none": "0",
+            "zero": "0",
+            "one": "1",
+            "two": "2",
+            "three": "3",
+            "four": "4",
+            "five": "5",
+            "six": "6",
+            "seven": "7",
+            "eight": "8",
+            "nine": "9",
+            "ten": "10",
+        }
+        self.articles = ["a", "an", "the"]
+        self.period_strip = re.compile("(?!<=\d)(\.)(?!\d)")
+        self.comma_strip = re.compile("(?<=\d)(\,)+(?=\d)")
+        self.punct = [
+            ";",
+            r"/",
+            "[",
+            "]",
+            '"',
+            "{",
+            "}",
+            "(",
+            ")",
+            "=",
+            "+",
+            "\\",
+            "_",
+            "-",
+            ">",
+            "<",
+            "@",
+            "`",
+            ",",
+            "?",
+            "!",
+        ]
 
-def get_score(occurences):
-    if occurences == 0:
-        return 0
-    elif occurences == 1:
-        return 0.3
-    elif occurences == 2:
-        return 0.6
-    elif occurences == 3:
-        return 0.9
-    else:
-        return 1
+    def word_tokenize(self, word):
+        word = word.lower()
+        word = word.replace(",", "").replace("?", "").replace("'s", " 's")
+        return word.strip()
 
+    def process_punctuation(self, inText):
+        outText = inText
+        for p in self.punct:
+            if (p + " " in inText or " " + p in inText) or (
+                re.search(self.comma_strip, inText) is not None
+            ):
+                outText = outText.replace(p, "")
+            else:
+                outText = outText.replace(p, " ")
+        outText = self.period_strip.sub("", outText, re.UNICODE)
+        return outText
 
-def process_punctuation(inText):
-    outText = inText
-    for p in punct:
-        if (p + " " in inText or " " + p in inText) or (
-            re.search(comma_strip, inText) is not None
-        ):
-            outText = outText.replace(p, "")
-        else:
-            outText = outText.replace(p, " ")
-    outText = period_strip.sub("", outText, re.UNICODE)
-    return outText
+    def process_digit_article(self, inText):
+        outText = []
+        tempText = inText.lower().split()
```

Comment:
Please check for snake_case everywhere.

Reply:
I am aware that old code was also camelCase but it is right time to update this to be snake_case